desc/compute/geom_utils.py

```diff
@@ -53,6 +57,45 @@ def rotation_matrix(axis, angle=None):
     return jnp.where(norm < eps, jnp.eye(3), R1 + R2 + R3)  # if axis=0, no rotation
 
 
+def _skew_matrix(a):
+    return jnp.array([[0, -a[2], a[1]], [a[2], 0, -a[0]], [-a[1], a[0], 0]])
+
+
+def rotation_matrix_vector_vector(a, b):
+    """Matrix to rotate vector a onto b.
+
+    NOTE: not correct if a and b are antiparallel, will
+    simply return identity when in reality negative identity
+    is correct.
+
+    Parameters
+    ----------
+    a,b : array-like, shape(3,)
+        Vectors, in cartesian (X,Y,Z) coordinates
+        Matrix will correspond to rotating a onto b
+
+
+    Returns
+    -------
+    rotmat : ndarray, shape(3,3)
+        Matrix to rotate points in cartesian (X,Y,Z) coordinates.
+
+    """
+    a = jnp.asarray(a)
+    b = jnp.asarray(b)
+    a = safenormalize(a)
+    b = safenormalize(b)
+    axis = cross(a, b)
+    norm = safenorm(axis)
+    axis = safenormalize(axis)
+    eps = 1e2 * jnp.finfo(axis.dtype).eps
+    skew = _skew_matrix(axis)
+    R1 = jnp.eye(3)
+    R2 = skew
+    R3 = (skew @ skew) * safediv(1, 1 + dot(a, b))
+    return jnp.where(norm < eps, jnp.eye(3), R1 + R2 + R3)  # if axis=0, no rotation
```

Comment:
Not sure this is correct, I seem to get weird answers in a few cases I've tried. 
The determinant of a rotation matrix should always be 1, but for example

```python
np.linalg.det(rotation_matrix_vector_vector(np.array([1,0,0]), np.array([1,1,0])))
```
gives 1.17

and if they are almost antiparallel:
```python
np.linalg.det(rotation_matrix_vector_vector(np.array([1,0,0]), np.array([-1,1e-4,0])))
```
gives like 4e16